Frontend/src/GenericComponents/SVGLogos/SVGLogos.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+
+const logosImages = [
+    {
+        type: "Stop",
+        image: <svg xmlns="http://www.w3.org/2000/svg" className="m-auto h-14 w-14" fill="none" viewBox="0 0 26 26" stroke="#868EBB" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
```

Comment:
En react vos mieux utiliser strokeWidth au lieu de strokeWidth. A verifier dans les autres SVG
```suggestion
        image: <svg xmlns="http://www.w3.org/2000/svg" className="m-auto h-14 w-14" fill="none" viewBox="0 0 26 26" stroke="#868EBB" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round">
```